appveyor.yml

```diff
@@ -1,30 +1,39 @@
-version: 0.{build}
+version: 1.0.{build}
+os: Visual Studio 2017
+clone_depth: 2
+configuration: Release
 
-image: Visual Studio 2017
+dotnet_csproj:
+  patch: false
+  file: '**\*.csproj'
+  version: '{version}'
+  package_version: '{version}'
+  assembly_version: '{version}'
+  file_version: '{version}'
+  informational_version: '{version}'
 
 pull_requests:
   do_not_increment_build_number: true
-# scripts to run before build
-before_build: 
- - nuget restore
- - dotnet restore
-configuration: Release
-build:
-  project: Minio.sln
-  publish_nuget: true
-  publish_nuget_symbols: true
-
-test:
-  # only assemblies to test
-  assemblies:
-    only:
-      - '**\*.Tests.dll'
-
-assembly_info:
-  patch: true
-  file: AssemblyInfo.*
-  assembly_version: "{version}"
-  assembly_file_version: "{version}"
-  assembly_informational_version: "{version}"
+
+environment:
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+  DOTNET_CLI_TELEMETRY_OPTOUT: 1
+
+  # scripts to run before build
+before_build:
+- appveyor-retry dotnet restore -v Minimal
+
+build_script:
+  - ps: dotnet build -c Release
+
+after_build:
+  - ps: dotnet pack .\Minio\Minio.csproj --no-build -c Release -o .\artifacts
```

Comment:
@poornas all you gotta do is run this command and it will automatically package the nuget packages on any platform (osx, windows, linux)

Reply:
ok.